### ISSUE TITLE
Document description is inconsistent with interface specification

### ISSUE THREAD

**Issue by lost22git** [fubark/cyber#93]
![Screenshot 2024-06-16 162843](https://github.com/fubark/cyber/assets/65008815/46c25260-af59-4b0d-bc8f-5e95e5b5ceae)

**Comment by fubark** [comment]
Correct, these functions now throw errors instead. The docs should reflect that.
